README.md

```diff
@@ -1,2 +1,92 @@
-# Hello-Microverse
-Understanding Git and GitHub flow
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Hello World README file</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello World] <a name="about-project"></a>
+
+**[Hello Microverse]** is a simple exercise of the concepts we have learned so far. Concepts such as git, GitHub, GitHub flow and other. Just think of it as a practice run for me, as a new micronaut.
+
+## 🛠 Built With <a name="Git"></a>
+<p>I made use of Git and GitHub.</p>
+<p>I also made use of simple html and CSS, as required of me</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+https://github.com/Chimoneg27/Hello-Microverse/pull/1 <!--This is just a link to my original pull request-->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+There is nothing much in this project. As mentioned in the about section. This project is for understanding the concepts of git, GitHub, GitHub flow and many other features.
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Garvin**
+
+- GitHub: [@Chimoneg27](https://github.com/Chimoneg27)
+- Twitter: [@Chimoneg_27](https://twitter.com/Chimoneg_27)
+- LinkedIn: [Garvin Chimone](https://www.linkedin.com/in/garvin-chimone-37208924a/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```

Comment:
Great work @Chimoneg27 , just a small last thing to fix before the approval, kindly remove the default text under each section to make your work look clean and professional

Reply:
@souad988. It's all fixed. You may check the code